### PR TITLE
Config files to use iFeel retargeting added for iCubGenova04

### PR DESCRIPTION
**Pull request by isorrentino**

### CODE REVIEW

File: src/WalkingModule/app/robots/iCubGenova04/dcm_walking_iFeel_joint_retargeting.ini [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+name                    walking-coordinator
+# height of the com
+com_height              0.53
+# sampling time
+sampling_time           0.01
+# enable joint retargeting
+use_joint_retargeting    1
+# enable the virtualizer
+use_virtualizer         1
+use_com_retargeting     1
```
Comment (S-Dafarra) [inline src/WalkingModule/app/robots/iCubGenova04/dcm_walking_iFeel_joint_retargeting.ini:25]: I think this needs to be zero

Comment (GiulioRomualdi) [reply src/WalkingModule/app/robots/iCubGenova04/dcm_walking_iFeel_joint_retargeting.ini:25]: it depends if iFeel provides the CoM position. @kouroshD, did you disable the port for iFeel?

If I'm not mistaken, you (@S-Dafarra ) told me that with this flag on the walking was failing at startup. If yes, I've no problem apply the following suggestion
```suggestion
use_com_retargeting     0
```

Comment (S-Dafarra) [reply src/WalkingModule/app/robots/iCubGenova04/dcm_walking_iFeel_joint_retargeting.ini:25]: Yes, it was failing. I did not investigate too deeply who was supposed to open that port.

**Comment by GiulioRomualdi** [comment]
In https://github.com/robotology/walking-controllers/pull/87/commits/42870dd1e9eb7827749e3271db3644cd4810e8c0 I disabled `use_com_retargeting` when iFeel is enabled.

@S-Dafarra let me know if I can merge it :)

**Comment by S-Dafarra** [comment]
> In [42870dd](https://github.com/robotology/walking-controllers/commit/42870dd1e9eb7827749e3271db3644cd4810e8c0) I disabled `use_com_retargeting` when iFeel is enabled.
> 
> @S-Dafarra let me know if I can merge it :)

The modifications of https://github.com/robotology/walking-controllers/pull/87/commits/49431bd819f8c4372a46d92f0c173923cd3ed99e were temporary? The ``r_wrist_pronosup`` has been removed because it was broken. Do we want to keep this by default?

**Comment by GiulioRomualdi** [comment]
Working on it :)

**Comment by GiulioRomualdi** [comment]
I actually noticed that https://github.com/robotology/walking-controllers/commit/49431bd819f8c4372a46d92f0c173923cd3ed99e was required because iFeel was streaming only the following joints

https://github.com/robotology/walking-controllers/blob/42870dd1e9eb7827749e3271db3644cd4810e8c0/src/WalkingModule/app/robots/iCubGenova04/dcm_walking/iFeel_joint_retargeting/jointRetargeting.ini#L6-L14

As far as I know, now iFeel streams also the wrist information. If yes I can reenable the wrists  

cc @kouroshd @lrapetti

**Comment by kouroshD** [comment]
@GiulioRomualdi Yes, as you can see in https://github.com/robotology/walking-teleoperation/blob/devel/app/robots/iCubGenova09/iFeelRetargetingWalking.ini, we are sending all the wrist joints.

**Comment by S-Dafarra** [comment]
> @GiulioRomualdi Yes, as you can see in https://github.com/robotology/walking-teleoperation/blob/devel/app/robots/iCubGenova09/iFeelRetargetingWalking.ini, we are sending all the wrist joints.

Note that we are referring to iCubGenova04

**Comment by GiulioRomualdi** [comment]
For iCubGenova04 the wrists are disabled

https://github.com/robotology/walking-teleoperation/blob/b27616c51758176bac140a8469c5d5fd6e16fdb9/app/robots/iCubGenova04/iFeelRetargetingWalking.ini#L15-L17

**Comment by kouroshD** [comment]
> Note that we are referring to iCubGenova04
OK, so we should add them to the iCubGenova04 walking teleoperation which are missing.

**Comment by kouroshD** [comment]
> I opened a small pr on walking teleoperation side to fix this. robotology/walking-teleoperation#64

@isorrentino  The problem on walking teloperation side has been fixed. feel free to apply the suggested fix here and merge the pr if there are no other revision here.

**Comment by isorrentino** [comment]
This commit https://github.com/robotology/walking-controllers/pull/87/commits/9c2b2c454ae148511f588615edbdc1bb1237e65e adds the missing wrist joints for `iCubGenova04` and  `iCubGazeboV2_5`. @kouroshD @S-Dafarra

**Comment by GiulioRomualdi** [comment]
Why this commits? 
image

**Comment by GiulioRomualdi** [comment]
Furhermore you deleted 42870dd1e9eb7827749e3271db3644cd4810e8c0 and 49431bd819f8c4372a46d92f0c173923cd3ed99e

This is the right history
```
git log
commit 42870dd1e9eb7827749e3271db3644cd4810e8c0 (HEAD -> walking_with_iFeel_Retargeting, origin/walking_with_iFeel_Retargeting, origin.20210616110407/walking_with_iFeel_Retargeting)
Author: Giulio Romualdi <giulio.romualdi@gmail.com>
Date:   Thu Jul 1 10:39:24 2021 +0200

    Disable com retargeting when iFeel retargeting is enabled

commit 49431bd819f8c4372a46d92f0c173923cd3ed99e
Author: Giulio Romualdi <giulio.romualdi@gmail.com>
Date:   Fri May 28 16:36:00 2021 +0000

    Update the configuration files for iFeel retargeting

commit 2302de825186749588319bdc1bd620a3e971f97b
Author: Ines <ines.sorrentino@iit.it>
Date:   Fri May 28 11:41:36 2021 +0200

    Config files to use iFeel retargeting added for iCubGenova04
```

Try to avoid to do `push --force` if you are not 100% sure 😸

**Comment by isorrentino** [comment]
@GiulioRomualdi I noticed that the branch was not aligned with the devel branch. I just did the rebase, the commit of the modifications for the two files and checked the log before doing `push --force`. I probably can’t read anymore, next time I’ll be more careful, I’m sorry.

**Comment by GiulioRomualdi** [comment]
Do not worry @isorrentino :) I've just fixed it.

Now I will add your commit

**Comment by GiulioRomualdi** [comment]
The PR can be now meged. Waiting your (@isorrentino @S-Dafarra and @kouroshD) green flag.

Could you please link the commit where the joints  have been enabled in the teleoperation side?

**Comment by kouroshD** [comment]
> Could you please link the commit where the joints have been enabled in the teleoperation side?

here is the PR: https://github.com/robotology/walking-teleoperation/pull/64 and the commit number https://github.com/robotology/walking-teleoperation/commit/c7d0c791333698db84541e3501fc111540d7d591

**Comment by GiulioRomualdi** [comment]
Top @kouroshD

**Review by S-Dafarra — APPROVED** [review]

**Pull request merged**